app/controllers/pages_controller.rb

```diff
@@ -5,6 +5,10 @@ class PagesController < ApplicationController
   # GET /pages.json
   def index
     @pages = Page.all
+
+    @pages.find &:blank?
+
+    @pages.detect &:blank?
```

Comment:
Ambiguous block operator. Parenthesize the method arguments if it's surely a block operator, or add a whitespace to the right of the `&` if it should be a binary AND.